src/main/kotlin/no/nav/familie/ef/sak/config/RestTemplateConfig.kt

```diff
@@ -0,0 +1,95 @@
+package no.nav.familie.ef.sak.config
```

Comment:
Litt usikker på om disse trengs, dette blir konfiguert i felles
RestTemplateBuilderBean
RestTemplateAzure
RestTemplateBuilderBean

Eller savner de noe spesielt?

Reply:
Ja, de i felles bruker proxy by default. Det kan vi ikke gjøre lokalt med preprod-konfig, derav if-sjekken på Spring-profiler som jeg nevnte i PR-beskrivelsen. Den sjekken brukes både i RestTemplateBuilder-bønna og i RestOperations-bønna.

Reply:
Virker ikke dette da gjennom at NaisNoProxyCustomizer finnes i test-scope ?

Reply:
Mulig det vil fungere, men jeg syns det er mye ryddigere å konfigurere opp RestTemplateBuilder og RestOperations slik vi vil ha de i appen vår i stedet for å ha en mer eller mindre vilkårlig fil i test-scope. Det er veldig vanskelig å se hva NaisNoProxyCustomizer brukes til om man setter den inn i appen sånn, den injectes jo i en komponent i et utenforstående bibliotek...